sql/server.go

```diff
@@ -226,9 +270,10 @@ func (s *Server) exec(req driver.Request) (resp driver.Response, err error) {
 		if err = plan.Err(); err != nil {
 			return
 		}
-
 		resp.Results = append(resp.Results, result)
 	}
-
+	if err != nil {
+		panic(err)
```

Comment:
Not sure what you are suggesting here. FYI if you explicitly return values you cannot change them in a defer (unless they are pointers pointing to a value)